### PR TITLE
Added entry for TowerForge enableGPU

### PR DESCRIPTION
**Pull request by gwright99**

### CODE REVIEW

**Comment by gwright99** [comment]
+ @tcrespog @swampie for accuracy assessment from a Tower Forge perspective

File: docs/faqs.md [hunk 1]
```diff
@@ -688,6 +688,19 @@ No. You can inject values directly into `tower.yml` or - in the case of a Kubern
...
 
+### Tower Forge
+
+**<p data-question>Q: What does the `Enable GPU` option do when building an AWS Batch cluster via Tower Forge?</p>**
+
+Activating the **Enable GPU** field while creating an AWS Batch environment with Tower Forge will result in an [AWS-recommended GPU-optimized ECS AMI](https://docs.aws.amazon.com/batch/latest/userguide/batch-gpu-ami.html) being used as your Batch cluster's default image. 
+
+Caveat!
+
+1. Activation does not cause GPU-enabled instances to automatically spawn in your Batch cluster. You must still specify these in the Forge screen's **Advanced option > Instance types** field.
```
Comment (llewellyn-sl) [inline docs/faqs.md:699]:
```suggestion
1. Activation does not cause GPU-enabled instances to automatically spawn in your Batch cluster. You must still specify these in the Forge screen's **Advanced options > Instance types** field.
```

File: docs/faqs.md [hunk 2]
```diff
@@ -688,6 +688,19 @@ No. You can inject values directly into `tower.yml` or - in the case of a Kubern
 Please contact Seqera Labs for more details if this is of interest.
 
 
+### Tower Forge
+
+**<p data-question>Q: What does the `Enable GPU` option do when building an AWS Batch cluster via Tower Forge?</p>**
+
+Activating the **Enable GPU** field while creating an AWS Batch environment with Tower Forge will result in an [AWS-recommended GPU-optimized ECS AMI](https://docs.aws.amazon.com/batch/latest/userguide/batch-gpu-ami.html) being used as your Batch cluster's default image. 
+
+Caveat!
```
Comment (llewellyn-sl) [inline docs/faqs.md:697]:
```suggestion
Note:
```

**Review by llewellyn-sl — APPROVED** [review]

**Review by abhi18av — APPROVED** [review]
LGTM ✅

**Review by pditommaso — APPROVED** [review]

**Pull request merged**